style/main.css

```diff
@@ -0,0 +1,814 @@
+  justify-content: center;
+}
+
+.about-member .name {
+  font-size: 2.3rem;
+  color: var(--textColor);
+  margin-bottom: 1rem;
+  text-align: center;
+}
+
+.about-member i {
+  color: var(--secondaryColor);
+  font-weight: 600;
+  margin-bottom: 1rem;
+  position: relative;
+}
+
+.about-member i::after {
+  position: absolute;
+  content: '';
+  height: 2px;
+  width: 25px;
+  background-color: var(--primaryColor);
+  bottom: -1.2rem;
+  left: 0;
+  z-index: 100;
+}
+
+.about-member .member-detail {
+  margin-top: 1.7rem;
+  font-size: 1.4rem;
+  font-weight: 900;
+  color: var(--textColor);
+}
+
+.team .see-more {
+  width: 100%;
+  text-align: center;
+  margin-bottom: 3rem;
+}
+
+.team .see-more .btn {
+  display: inline-block;
+  width: 80%;
+  margin: 0 auto;
+  padding: 1.6rem 0;
+  background-color: #fff;
+  border: 1px solid var(--primaryColor);
+  text-transform: uppercase;
+  font-size: 1.7rem;
+  text-align: center;
+  color: var(--textColor);
+}
+
+.team .see-more .btn:hover {
+  background-color: var(--secondaryColor);
+  color: var(--textColor);
+  font-weight: 700;
+}
+
+.our-partners {
+  width: 100%;
+  background-color: var(--textColor);
+  opacity: 0.9;
+  padding: 3rem 0;
+  margin-top: 3rem;
+}
+
+.about-title {
+  width: 70%;
+  text-align: center;
+  position: relative;
+  margin: 3rem auto;
+  color: var(--textColor);
+}
+
+.about-title::after {
+  position: absolute;
+  content: '';
+  height: 1px;
+  width: 30px;
+  background-color: var(--secondaryColor);
+  bottom: -1.4rem;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 100;
+}
+
+.our-partners .about-title {
+  color: #fff;
+}
+
+.partner-list {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.partner-list li {
+  margin: 1rem 1rem;
+}
+
+.team .see-more .btn span:last-child {
+  color: var(--secondaryColor);
+  margin-left: 1rem;
+}
+
+.team .see-more .btn:hover span:last-child {
+  color: var(--primaryColor);
+}
+
+#footer {
+  width: 100%;
+}
+
+.footer-main-container {
+  width: 80%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 5rem 0;
+}
+
+#footer .logo {
+  height: 6.5rem;
+  width: 15rem;
+  margin-right: 2.5rem;
+}
+
+#footer .logo img {
+  max-width: 100%;
+}
+
+#footer .created {
+  display: flex;
+  justify-content: flex-end;
+}
+
+#footer .created p {
+  color: var(--textColor);
+}
+
+@media screen and (min-width: 500px) {
+  .team-members {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 3rem;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .top-navbar {
+    display: block;
+  }
+
+  .main-navbar {
+    box-shadow: 0 8px 16px rgb(39, 42, 49, 0.3);
+    background-color: #fff;
+    padding: 1rem 0;
+  }
+
+  .main-navbar .navbar .menu {
+    display: none;
+  }
+
+  .main-navbar .logo {
+    display: block;
+  }
+
+  .navbar .nav-list {
+    display: flex;
```

Comment:
- [ ] Please use DRY rule in your CSS files you can group repeated rules in one class to avoid repeating yourself like `display:flex` is repeated many times check it out and check all css files to see where to improve